gemma-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -163,7 +177,7 @@
         </init-param>
         <init-param>
             <param-name>jersey.config.server.provider.classnames</param-name>
-            <param-value>org.glassfish.jersey.message.GZipEncoder</param-value>
+            <param-value>org.glassfish.jersey.message.GZipEncoder,org.glassfish.jersey.server.spring.scope.RequestContextFilter</param-value>

```

Comment:
The `RequestContextFilter` does not work nicely with tests, but is necessary for using the `RequestAttributesHolder`.

Reply:
Turns out we don't need use the servlet filter any longer and we can mock the AnalyticsProvider in tests.